### DISCUSSION TITLE
Install - it wants me to make a Sophie's choice?

### DISCUSSION THREAD

**Discussion by romanr** [matt8707/ha-fusion#323] [Q&A]
I clicked "install in my HA" and then this dialog appears. No explanation, no buttons.

 I suppose it want me to make choice and kill Zigbee2MQTT as a sacrifice?

<img src="https://github.com/matt8707/ha-fusion/assets/329079/72ff44c8-e7a0-44ad-a4bf-7b5e876f1cd1" width=500>

What does it mean? 
I try very hard to concentrate and read it's mind. But nothing comes through. Must be Zigbee signal blocking.

(using supervised HA OS, latest versions)

**Comment by matt8707** [comment]
No, you've added repo in step 1, now do step 2

1. **Add Repository**: To begin, add the ha-fusion add-on repository to your Home Assistant instance. Click the button below or manually add the repository using this URL: <https://github.com/matt8707/addon-ha-fusion>.

2. **Install Add-on**: After adding the repository, refresh the add-on store page. Locate ha-fusion in the list and proceed with the installation.
